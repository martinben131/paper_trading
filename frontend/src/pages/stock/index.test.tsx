import React from "react";
import { shallow } from "enzyme";
import Stock from "./index";

test("Renders stock correctly", () => {
  const component = shallow(<Stock />);
  expect(component).toMatchSnapshot();
});
