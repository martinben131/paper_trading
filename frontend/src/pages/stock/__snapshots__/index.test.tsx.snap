// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders stock correctly 1`] = `
<Fragment>
  <div
    className="firstRow"
  >
    <div
      className="productDetails"
    >
      <Chart />
    </div>
    <div
      className="buySellContainer subtitleHeading"
    >
      <Trade />
    </div>
  </div>
  <div
    className="newsSection"
  >
    <News />
  </div>
</Fragment>
`;
